perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -157,6 +161,18 @@ def RobustRemoteCommand(self, command):
                     '--delete']
     return self.RemoteCommand(' '.join(wait_command), should_log=False)
 
+  def SetupRemoteFirewall(self):
+    """Sets up IP table configurations on the VM."""
+    if hasattr(self, 'allowed_ip_range') and self.allowed_ip_range:
```

Comment:
This way is more secure of course. And other cloud providers could easily do that just by passing allowed_ip_range = everything in their virtual_machine classes